canopeum_backend/canopeum_backend/views.py

```diff
@@ -902,15 +906,14 @@ def post(self, request: Request):
         except json.JSONDecodeError as e:
             return Response(data={"error": e}, status=status.HTTP_400_BAD_REQUEST)
 
-        # HACK to allow handling the image with a AssetSerializer separately
-        # TODO: Figure out how to feed the image directly to BatchDetailSerializer
-        image = None
-        if request.data.get("image"):
-            asset_serializer = AssetSerializer(data=request.data)
+        uploaded_images = request.FILES.getlist("images", [])
```

Comment:
https://github.com/BesLogic/releaf-canopeum/pull/354/files#r2094244259
> This is still a hack and this should still be the job of a serializer/deserializer. We shouldn't have to write this in views, even less so for ever view that might need it.

```suggestion
        # HACK to allow handling the image with a AssetSerializer separately
        # TODO: Figure out how to feed the image directly to BatchDetailSerializer
        uploaded_images = request.FILES.getlist("images", [])
```